.gitignore

```diff
@@ -4,6 +4,7 @@
 **/target
 **/*.rs.bk
 .idea/
+Cargo.lock
```

Comment:
we can't do this, need it to ensure the build consistency

Reply:
+1. If you feel differently, I'd recommend making a separate PR where folks can discuss 😄